app/assets/javascripts/modules/Filter.js

```diff
@@ -0,0 +1,10 @@
+define(['jquery', 'List', 'ListFuzzySearch'], function($, List, ListFuzzySearch) {
+  'use strict';
+
+  var firmOptions = {
+    valueNames: ['js-frn', 'js-firm-name', 'js-principal-name'],
+    plugins: [ListFuzzySearch()]
+  };
+
+  var firmList = new List('firm-list', firmOptions);
```

Comment:
And I assume the List object can't get garbage collected because it attaches itself to the DOM? Worth checking cause things won't get garbage collected immediately and different browsers behave differently right?